pyengine/katana.py

```diff
@@ -55,4 +55,12 @@ def matcher(qty,book):
     for exslice in slices:
         print(exslice)
 
-    return slices
\ No newline at end of file
+    return slices
+
+def quotetrimmer(limitprice,book):
+    for quote in book.copy():
```

Comment:
why do we need book.copy()?

Reply:
i was running into some problems iterating through a dictionary that i was actively making changes to so i made a copy.

we use the copy to go process each quote, but then remove the quotes from the original dictionary